src/style.css

```diff
@@ -1,3 +1,74 @@
+
 body {
   margin: 0;
+  background-color: #e5e5e5;
+  display: flex;
+  justify-content: center;
+}
+
+.header {
+  max-width: 1200px;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.logo__img {
+  margin-top: 10px;
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: inline-block;
```

Comment:
The 'display: inline-block;' is redundant as the parent container (.nav) is not a flex container. You can safely remove this line.